### ISSUE TITLE
feat: For a ML task, include some baselines

### ISSUE THREAD

**Issue by sylvaincom** [probabl-ai/skore#1025]
### Is your feature request related to a problem? Please describe.

Data scientists often rush to use relatively complex models on a new data set they discover, without trying out some simple baselines.

### Describe the solution you'd like

Add some baseline models.

**Comment by glemaitre** [comment]
There is stuff before to do LLM-assisted stuff: https://skrub-data.org/stable/reference/generated/skrub.tabular_learner.html

**Comment by sylvaincom** [comment]
Yes! I was also thinking about the [dummy estimators](https://scikit-learn.org/1.5/api/sklearn.dummy.html) of scikit-learn?
